<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="5.85s"
		durationms="5847"
		type="root"
		rid="6b3eea6630ba2f6"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="E1238621"
		rxversion="9.3.4.0"
		host="INNOID0077"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="en-US"
		screenresolution="1920x1080"
		timestamp="29-Jan-21 11:39:26 AM"
		timestampiso="2021-01-29T06:09:26"
		endtime="29-Jan-21 11:39:31 AM"
		timeoutfactor="1"
		runid=""
		sutversion="">
		<detail>
			Ranorex Testscript
		</detail>
		<item
			timeRelativeToTestSuiteStartTime="00:05.848"
			timeRelativeToTestModuleStartTime="00:05.848"
			timeWallClock="11:39:31"
			time="00:05.848"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
